Rakefile

```diff
@@ -6,7 +6,7 @@ require 'rubocop/rake_task'
 
 Rails.application.load_tasks
 RuboCop::RakeTask.new
-task default: [:create_reports_dir, :rubocop, 'cucumber:html', 'brakeman:run', 'bundle_audit:run',
+task default: [:create_reports_dir, :rubocop, 'brakeman:run', 'bundle_audit:run',
```

Comment:
Nope, we are taking care of that in the Jenkins file by calling them on their own.  This way all the other tests can still run and we can get feedback if those fail, otherwise cucumber ends up being one of the first things run and fails before any of the other tests even run.